src/main/java/org/spongepowered/api/event/entity/AffectEntityEvent.java

```diff
@@ -73,6 +71,7 @@
      *
      * @return The List
```

Comment:
Just clarify what this list is returning, whether it's the original entities, the new entities affected, or the filtered entities that won't be affected.